src/graphics.c

```diff
@@ -237,28 +265,34 @@ void graphics_copy_to_color_buffer(int x, int y, int width, int height, struct g
 // Can't use graphics_line since it uses plot pixel.
 // It does mouse region checking, which draws into the mouse buffer.
 // It would then just draw into the mouse buffer, defeating the purpose of the mouse buffer.
-void graphics_draw_mouse() {
-    int i;
-    uint8_t *v;
+void graphics_draw_cross_mouse() {
+    // horizontal line
     int start_x = (mouse_x - MOUSE_SIDE_2 + 1) < 0 ? 0 : mouse_x - MOUSE_SIDE_2 + 1;
     int end_x = (mouse_x + MOUSE_SIDE_2) > video_xres ? video_xres : mouse_x + MOUSE_SIDE_2;
-    // horizontal line
-    for (i = start_x; i < end_x; i++) {
-        v = video_buffer + video_xbytes * mouse_y + i * 3;
-        v[2] = mouse_fg_color.r;
-        v[1] = mouse_fg_color.g;
-        v[0] = mouse_fg_color.b;
-    }
+    graphics_line(start_x, mouse_y, end_x - 1, mouse_y, mouse_fg_color, 1);
 
+    // vertical line
     int start_y = (mouse_y - MOUSE_SIDE_2 + 1) < 0 ? 0 : mouse_y - MOUSE_SIDE_2 + 1;
     int end_y = (mouse_y + MOUSE_SIDE_2) > video_yres ? video_yres : mouse_y + MOUSE_SIDE_2;
-    // vertical line
-    for (i = start_y; i < end_y; i++) {
-        v = video_buffer + video_xbytes * i + mouse_x * 3;
-        v[2] = mouse_fg_color.r;
-        v[1] = mouse_fg_color.g;
-        v[0] = mouse_fg_color.b;
-    }
+    graphics_line(mouse_x, start_y, mouse_x, end_y - 1, mouse_fg_color, 1);
+}
+
+// draws a standard pointer mouse
+void graphics_draw_pointer_mouse() {
+    // long pointer lines
+    // TODO: why does MOUSE_SIDE_2 - 3 work but not MOUSE_SIDE_2 - 1 not work (only draw line issue)
```

Comment:
I looked at this for a while and have no idea what's going on. I also talked with @kkoser about this, and we decided to just add a TODO.